libs/clients/syslumenn/src/lib/syslumennClient.service.ts

```diff
@@ -356,16 +354,12 @@ export class SyslumennService {
     const res = await api.vedbokavottordRegluverkiPost({
       skilabod: {
         audkenni: id,
-        fastanumer:
-          propertyNumber[0] == 'F'
-            ? propertyNumber.substring(1, propertyNumber.length)
-            : propertyNumber,
+        fastanumer: cleanPropertyNumber(propertyNumber),
         tegundAndlags: TegundAndlags.NUMBER_0, // 0 = Real estate
       },
     })
-
     if (res.length > 0) {
-      return {
+      const ret = {
```

Comment:
If the ret variable isn't used we could return the object immediately instead of assigning it to a variable and then returning it